numpy/core/src/common/array_assign.h

```diff
@@ -88,7 +88,8 @@ broadcast_strides(int ndim, npy_intp *shape,
 /*
  * Checks whether a data pointer + set of strides refers to a raw
  * array whose elements are all aligned to a given alignment.
- * alignment should be a power of two.
+ * alignment should be a power of two, or may be the sentinel value 0 to mean
+ * unaligned, in which case false is always returned.
```

Comment:
This comment confuses me - I read this as saying that `is_aligned(..., alignment = 0)` means `is_unaligned(...)`

Reply:
If I am not misreading you, that is what the comment is meant to say. If you pass an alignment of 0, the function always returns "not aligned". Do you mean I should say "not aligned" instead of "unaligned"?

Reply:
Right. the return value means "not aligned", but the meaning of the `alignment` argument is closer to "unalignable"

Reply:
fixing